plugins/aws/redshift/underutilizedRedshiftCluster.js

```diff
@@ -0,0 +1,89 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Underutilized Redshift Cluster Check',
+    category: 'Redshift',
+    domain: 'Databases',
+    description: 'Identify Redshift clusters that appear to be underutilized',
```

Comment:
Change to something like
```suggestion
    description: 'Ensure Redshift clusters are not being underutilized',
```